typings/index.d.ts

```diff
@@ -117,13 +117,13 @@ declare module 'discord.js' {
 
     public static create(
       target: MessageTarget,
-      content: APIMessageContentResolvable,
+      content: string | null,
       options?: undefined,
       extra?: MessageOptions | WebhookMessageOptions,
     ): APIMessage;
     public static create(
       target: MessageTarget,
-      content: StringResolvable,
+      content: string,
```

Comment:
Shouldn't this and the methods below until inclusive `reply` also allow `null`?

Reply:
not too sure what reply you are referring to but I think I applied the requested changes in https://github.com/discordjs/discord.js/pull/4880/commits/872675caab1da1ffb69394ddd8ae280f14798f83